mapactionpy_arcmap/MapChef.py

```diff
@@ -86,23 +99,29 @@ def cook(self, productName, countryName):
                                         arcpy.env.workspace = rasterFile
                                         rasters = arcpy.ListRasters("*")
                                         for raster in rasters:
-                                           if re.match(parts[1], raster):
```

Comment:
Again I do not understand how you are using the regexs here. Why you need to have forward slash in the regex, why you are then breaking the regex apart at the "/" and how you can ensure that each part of the regex will complie?